dockci/notifications.py

```diff
@@ -0,0 +1,31 @@
+""" Module for notification integration, for DockCI """
+import hypchat
+
+# pylint:disable=too-few-public-methods
+
+
+class HipChat():
+    """ HipChat Class """
+
+    def __init__(self, apitoken, room):
+        self.token = apitoken
+        self.roomname = room
+        self.roomid = 0
+        # probaly need this in a try catch
+        self.chatconn = hypchat.HypChat(apitoken)
+
+    def _verify_room(self):
```

Comment:
verify implies that this is a road only function, but it gets, then sets the room ID too. Maybe `_fill_room_info` or something?